.circleci/config.yml

```diff
@@ -93,22 +93,16 @@ workflows:
 
       # pre-staging
       - yarn/run:
-          name: test-jest-v1
+          name: test-jest
```

Comment:
lets rename this to just `test`

Reply:
There's already a `test` command I didn't want to conflict with (which is invoked in hokusai's test - which we _dont_ use on CI).